build/azure-pipelines/win32/ESRPClient/NuGet.config

```diff
@@ -1,6 +1,7 @@
 <?xml version="1.0" encoding="utf-8"?>
 <configuration>
     <packageSources>
+        <clear />
```

Comment:
Is this something new?

Reply:
This is the nuget check that I've discussed in the DRI meeting. Without it pipelines will fail